install/windows/config.txt

```diff
@@ -3,6 +3,8 @@ dir = C:\MobSF
 subdir_downloads = C:\MobSF\Downloads\

```

Comment:
Is this config applicable only for the Windows VM or for the Host Windows Machine as well?
If so we need to make if similar to 
https://github.com/ajinabraham/Mobile-Security-Framework-MobSF/blob/master/MobSF/settings.py#L50